bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -150,7 +150,11 @@ def _bytes_to_json(value):
 def _timestamp_to_json(value):
```

Comment:
If we promote it to `core`, maybe it needs a more descriptive name (probably something about the actual format, rather than the API).  As it is, the fact that it is in `bigquery/_helpers.py` should be enough.